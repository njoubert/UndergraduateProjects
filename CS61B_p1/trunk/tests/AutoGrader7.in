Stefan is the geneticist.
Alex is not the geneticist.
The businesswoman lives around here.
There is a buzzard house.
Who lives in the buzzard house?
What does Alex do?
Who is the geneticist?

