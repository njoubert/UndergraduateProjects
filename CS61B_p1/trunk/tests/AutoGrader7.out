1. Stefan is the geneticist.
2. Alex is not the geneticist.
3. The businesswoman lives around here.
4. There is a buzzard house.
Q: Who lives in the buzzard house?
A: I don't know.
Q: What does Alex do?
A: Alex is the businesswoman.
Q: Who is the geneticist?
A: Stefan is the geneticist.
